use-cases/device-integrations.md

```diff
@@ -15,3 +15,20 @@ In case your device is a Fiscal Machine \(no matter whether it is a virtual or a
 #### Key cutter commands
 
 In case your device is a Key Cutter, you would get a command containing [Key cutter command data](../operations/integrations.md#key-cutter-command-data).
+
+### Testing your integration
+
+Once your integration is completed, all endpoints should be tested prior to initiating the [certification process](https://intercom.help/mews-systems/en/articles/4497819-connector-api-certification-what-to-expect) with the Mews Marketplace team. Testing your solution is done directly in the Connector API demo. You should use the credentials found in the [Authentication](../guidelines.md#authentication) section to sign in as an end-user. This allows you to fully understand how both systems will be used by the property and gives you additional information to ensure you provide a seamless and efficient onboarding experience for our hoteliers.
+
+To be able to use your device integration, an additional step of configuring the device and connecting it to the relevant API access token is required. This is done through the following steps:
+
+1. Open the main menu in the top left corner, scroll down and click on **Marketplace**.
+2. Depending on the type of device you are connecting, click on either Facility management (Key cutter integration, Passport scanner integration, Printer integration) or Legal environment (Fiscal machine integration) categories.
+3. Once you've found the correct device, click **See more** followed by **Connect** to generate the new device.
+4. Complete the required fields for the relevant device and be sure to save the configuration.
+5. Go back to your unique API access token found in the **My subscriptions** section of the Marketplace and click on the **Settings** found on the right side of the integration listing.
```

Comment:
Wouldn't it be better to say "Go back to your unique integration client", since we're always referring to the integration or integration client? 

Then the Access Token is one element in that particular integration?

Reply:
I'm afraid that they then associate it with Client token and it leads to the incorrect switching between the two in the requests

Reply:
I would say rather just simply:
Go back to your connected integration ? @stang704 ? @czech-man ?